### DISCUSSION TITLE
How do I override the default `<C-s>` mapping?

### DISCUSSION THREAD

**Discussion by txtyash** [LazyVim/LazyVim#1261] [Q&A]
I'm trying to override the default `<C-s>` mapping by:
```lua
map({ "i" }, "<C-s>i", "<cmd>IconPickerInsert<cr><esc>", { remap = true, desc = "Save file" })
```
This mapping does not work probably because `<C-s` is already mapped by Lazyvim. How do I override this?

**Comment by dpetka2001** [comment]
`vim.keymap.del("i", "<C-s>")`. This will just disable `<C-s>` in `insert` mode, so you can use your own mapping in `insert` mode and you can still use `<C-s>` for saving file in other modes.
